server/models/Articles.js

```diff
@@ -0,0 +1,9 @@
+
+const nameSchema = new mongoose.Schema({
```

Comment:
This should be called "articleSchema", because it's a schema for an article, not a name.